package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/http-client",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I would just bump the patch version.  we want folks to get these changes and it's part of rounding out what 1.0.x offered